docs/static/settings/configuration-management-settings.asciidoc

```diff
@@ -54,8 +54,8 @@ section in your Logstash configuration, or a different one. Defaults to
 If your {es} cluster is protected with basic authentication, these settings
 provide the username and password that the Logstash instance uses to
 authenticate for accessing the configuration data. The username you specify here
-should have the built-in `logstash_admin` role and the customized `logstash_writer` role, which provides access to system
-indices for managing configurations. Starting with Elasticsearch version 7.10.0, the
+should have the built-in `logstash_admin` and `logstash_system` roles.
+These roles provide access to system indices for managing configurations. Starting with Elasticsearch version 7.10.0, the
 `logstash_admin` role inherits the `manage_logstash_pipelines` cluster privilege for centralized pipeline management.
```

Comment:
Let's keep the statements consistent as much as possible.

```suggestion
These roles provide access to system indices for managing configurations.

NOTE: Starting with Elasticsearch version 7.10.0, the `logstash_admin` role inherits the `manage_logstash_pipelines` cluster privilege for centralized pipeline management.
```

Reply:
I want to keep the sentence as part of the paragraphs because it provides mandatory information for 7.x, rather than serving as a supportive message